src/mlpack/methods/det/CMakeLists.txt

```diff
@@ -4,11 +4,7 @@
 set(SOURCES
   # the DET class
   dtree.hpp
-  dtree.cpp
-
-  # the util file
-  dt_utils.hpp
-  dt_utils.cpp
+  dtree_impl.hpp
```

Comment:
Shouldn't we include `dt_utils.hpp` and `dt_utils_impl.hpp` too?  I overlooked this in my review :)